src/index.html

```diff
@@ -1,57 +1,28 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link
-      rel="stylesheet"
-      href="https://pro.fontawesome.com/releases/v5.15.1/css/all.css"
-    />
-    <link rel="stylesheet" href="style.css" type="text/css" />
-    <title>To-Do-List</title>
+    <meta charset="UTF-8">  
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<link href="style.css" rel="stylesheet">
+		<link rel="preconnect" href="https://fonts.googleapis.com">
```

Comment:
- [ ] Please make sure that checking item as completed is saved in the localStorage.
![image](https://user-images.githubusercontent.com/74744787/185767565-f90f80a6-c5db-484b-8234-c2a0f0692a0b.png)